src/Viewer.tsx

```diff
@@ -88,15 +87,13 @@ function Viewer(): ReactElement {
 
   const canv: CanvasOverlay = useConstructor(() => {
     const stateDeps = renderCanvasStateParamsSelector(useViewerStateStore.getState());
-    // TODO: Once Datasets can report whether they are 2D or 3D, CanvasOverlay
-    // should construct and swap between the two types of canvases on its own.
-    const use3DCanvas = true;
-    const innerCanvas = use3DCanvas ? new ColorizeCanvas3D(stateDeps) : new ColorizeCanvas2D();
-
-    const canvas = new CanvasOverlay(innerCanvas, stateDeps);
+    const canvas = new CanvasOverlay(stateDeps);
```

Comment:
CanvasOverlay now constructs and swaps between its own internal 2D and 3D canvas.